### ISSUE TITLE
RFC: Function to return contrast ratio between two colors -> contrastRatio()

### ISSUE THREAD

**Issue by brianespinosa** [styled-components/polished#302]
Currently I am not seeing a way to return the contrast ratio between two colors. I could technically use `getLuminance()` to return the two values separately, then subtract to get the difference. It seems like it would be helpful to be able to instead just pass two colors and get the contrast ratio returned.

**Use case:**
If I wanted to check a background color to make sure a foreground text color was dark/light enough to be legible, I could darken/lighten the foreground color to ensure legibility.

Basically this would be similar to `readableColor()` except you are not limited to only returning black/white based on the single color passed.

**Comment by bhough** [comment]
Added to V3 Roadmap #354

**Comment by brianespinosa** [comment]
Thanks @bhough 👍
